test/onnx/test_pytorch_onnx_caffe2.py

```diff
@@ -664,11 +664,14 @@ def test_maxpool2d_single_padding(self):
         model = nn.MaxPool2d(5, padding=2)
         self.run_model_test(model, train=False, batch_size=BATCH_SIZE)
 
-    @unittest.skip("C2 and PyTorch have small difference in padding implementation")
     def test_avgpool2d(self):
         model = nn.AvgPool2d(5, padding=(2))
```

Comment:
I think the default value of count_include_pad is True, we can set it as True in this case.

Reply:
Done.